services/transaction_record.proto

```diff
@@ -145,4 +146,21 @@ message TransactionRecord {
          */
         int32 prng_number = 20;
     }
+
+    /**
+     * Virtual address updated info, if virtual address is specified in the CryptoUpdate transaction. At most one update can be performed at a time.
+     */
+    VirtualAddressUpdate virtualAddressUpdate = 21;
+}
+
+message VirtualAddressUpdate {
+    /**
+     * An accounts 20-byte EVM address account identifier and nonce.
+     */
+    VirtualAddress virtualAddress = 1;
+
+    /**
+     * Flag if this address is being removed. If false virtual address is being added
+     */
+    bool removed = 2;
```

Comment:
Also we should make this nullable  since on virtualAddressUpdate.virtualAddress.isDefaultAddress updates this will be null.
```suggestion
    BoolValue remove = 2;
```